cmd/e3db/main.go

```diff
@@ -706,16 +712,28 @@ func cmdSignup(cmd *cli.Cmd) {
 	cmd.Spec = "[OPTIONS] [NAME] [EMAIL] [PASSWORD]"
 
 	cmd.Action = func() {
-		sdk, err := e3db.GetSDKV3(fmt.Sprintf(e3db.ProfileInterpolationConfigFilePath, *options.Profile))
+		sdk := e3db.ToznySDKV3{}
+		ctx := context.Background()
+		createdAccount, err := sdk.Register(ctx, *accountName, *accountEmail, *accountPassword, *apiBaseURL)
 		if err != nil {
 			dieErr(err)
 		}
-		ctx := context.Background()
-		createdAccount, err := sdk.Register(ctx, *accountName, *accountEmail, *accountPassword)
+		accountClientConfig := createdAccount.Account.Config
+		accountClientInfo := &e3db.ClientOpts{
```

Comment:
Does this struct have signing key fields? If so might be worth adding those in as well.

Reply:
It doesn't, but just for you I added them.